Tests/Util/AdminObjectAclDataTest.php

```diff
@@ -92,8 +90,6 @@ public function testSetForm()
      * @depends testSetForm
      *
      * @group legacy
-     *
-     * @deprecated
```

Comment:
I'm not sure: we will remove the code under test, which will make us notice this.